yt_dlp/extractor/youtube/_tab.py

```diff
@@ -2331,7 +2331,7 @@ class YoutubePlaylistIE(YoutubeBaseInfoExtractor):
     def suitable(cls, url):
         if YoutubeTabIE.suitable(url):
             return False
-        from ...utils import parse_qs
+        from yt_dlp.utils import parse_qs
```

Comment:
Not suitable to be applied here, but upstream avoids these imports for lazy extractors with this in `devscripts/make_lazy_extractors.py`
```py
module_contents = [
    module_template,
    get_source(InfoExtractor.suitable),
    get_source(InfoExtractor._match_valid_url) + '\n',
    'class LazyLoadSearchExtractor(LazyLoadExtractor):\n    pass\n',
    # needed for suitable() methods of Youtube extractor (see #28780)
    'from youtube_dl.utils import parse_qs, variadic\n',
]
```
If an extractor were to need some other utility function, one could decide whether to add the necessary import in the above fragment or in the extractor itself, according to the likelihood of other extractors using it.

Also see #12607.